pyproject.toml

```diff
@@ -33,6 +33,7 @@ classifiers = [
 [tool.poetry.dependencies]
 python = "^3.7"
 pytest = ">=3.6.3"
+ipwhois = "~=1.2.0"
```

Comment:
For libraries meant to be reused by many applications, strict dependencies are discouraged since they lock every dependent app to use that specific version and block them from upgrading to `1.4.0` for example.

We should be as lax as possible here, and as strict as possible in `shiphero_app`.
For example here `ipwhois = ">= 1.0.0"`

Reply:
[This](https://caremad.io/posts/2013/07/setup-vs-requirement/) article is a good reference to understand why. `setup.py` here would be our `pyproject.toml`

Reply:
Thanks, done. Good catch.